jumpscale/packages/marketplace/chats/wiki.py

```diff
@@ -19,7 +19,7 @@ def configuration(self):
         form.ask("Set configuration")
         self.user_email = self.user_info()["email"]
         self.envars = {
-            "TYPE": "wiki",
+            "TYPE": "Wiki",
```

Comment:
this env variable will be delegated to the flist, changing that will cause flist failure!

Reply:
also it was automated change from search and replace feature, sorry